Core/src/main/java/me/totalfreedom/plotsquared/PlotSquaredHandler.java

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
Can we get rid of the copyright notice? Given it's not their CopyRight...?

Reply:
on compile it throws an error at the very end saying we violate the license– given that we're open sourcing a fork i thought we should not violate their license then xd